go/vt/mysqlctl/backup_test.go

```diff
@@ -714,3 +713,26 @@ func TestShouldRestore(t *testing.T) {
 	assert.False(t, b)
 	assert.NoError(t, err)
 }
+
+func Test_scanLinesToLogger(t *testing.T) {
```

Comment:
Use CamelCase for function names, even `Test` ones.
```suggestion
func TestScanLinesToLogger(t *testing.T) {
```